recipes/pycompilation/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pycompilation" %}
+{% set version = "0.4.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycompilation-{{ version }}.tar.gz
+  sha256: fa8095aa956055b37de965d97bec0ba1a29d8b7354cc73bb4094d7286d8b883b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win or osx]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  files:
+    - on_the_fly_main.py
+  requires:
+    - {{ compiler('c') }}
+    - cython
+    - numpy
+    - pip
```

Comment:
Sorry for sending this to another round. Are `cython` and `pip` not `run` dependencies? Is this package functional at all without these?

Reply:
These are all technically optional dependencies.